utils/score/decay_score.go

```diff
@@ -29,12 +29,19 @@ func NewScoreStore(num, denom float64, inpTime time.Time) ScoreStore {
 // where now is the current time.
 //
 // Note that the returned ScoreStore has a new Time field set to the current time.
-func CalculateTimeDecayFunctionUpdate(oldScore, newScore ScoreStore, halfLife time.Duration, updateWeight float64, sampleTime time.Time) ScoreStore {
+func CalculateTimeDecayFunctionUpdate(oldScore, newScore ScoreStore, halfLife time.Duration, updateWeight float64, sampleTime time.Time, isHanging bool) (normalizedScoreStore ScoreStore, rawScoreStore ScoreStore) {
```

Comment:
remove isHanging Api from this calculation, the caller should use the previous raw value if it is hanging

Reply:
done. see https://github.com/lavanet/lava/commit/e39f68c9d92a766e2f838173f376a0dffde36406